packages/core-web/src/styles/scroll-top-button.css

```diff
@@ -0,0 +1,25 @@
+#scroll-top-button {
+    display: none;
+    position: fixed;
+    bottom: 20px;
+    right: 30px;
+    z-index: 99;
+    cursor: pointer;
+    opacity: 0.4;
+    transition-property: opacity;
+    transition-duration: 0.25s;
+}
+
+#scroll-top-button.lighten {
+    opacity: 0.15;
+}
+
+@media (hover: hover) {
```

Comment:
sorry missed this out, let's remove this, since we aren't doing any additional styling

Reply:
This is to resolve hover opacity styling being "stucked" on mobile view; or rather, on any devices that do not support hovering capability. Such devices will cause CSS styling for hover to "stick". 

More details in this website:
[https://css-tricks.com/solving-sticky-hover-states-with-media-hover-hover/](url)